matRad_progress.m

```diff
@@ -1,4 +1,4 @@
-function matRad_progress(currentIndex, totalNumberOfEvaluations)
+function matRad_progress(currentIndex, totalNumberOfEvaluations, cntBeam)
```

Comment:
I would not make any changes to this function to accomodate special cases of the dose calculation.

Lets include an additional output statement in the dose calculations (particles & photons) itself!

beam 1: progress  xxx%